envs/picking_env/multiple_subgoals/blocksworld-multi-subgoal-domain.pddl

```diff
@@ -0,0 +1,93 @@
+(define (domain blocksworld-multi-subgoal)
+    (:requirements :strips :equality :typing :negative-preconditions)
+    (:types 
+        location gripper box - object
+    )
+    (:predicates 
+        (object_at ?o - box ?loc - location)
+        (robot_at ?g - gripper ?loc - location)
+        (gripper_open ?g - gripper)
+        (holding ?g - gripper ?x - object)
+        (is_subgoal1 ?loc - location)
+        (is_subgoal2 ?loc - location)
+        (is_subgoal3 ?loc - location)
+        (is_goal ?loc - location)
+    )
+
+    (:action move-to-subgoal1
+        :parameters (?g - gripper ?from ?to - location)
+        :precondition (and
+            (not (is_goal ?from))
```

Comment:
I feel like we need negative preconditions for them to not be subgoal2 and subgoal3 as well. Similarly for other actions.